examples/webgl_buffergeometry_constructed_from_geometry.html

```diff
@@ -5,11 +5,6 @@
 		<meta charset="utf-8">
 		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
 		<link type="text/css" rel="stylesheet" href="main.css">
-		<style>
-			a {
-				color: #08f;
```

Comment:
I think these colors were re-defined because the link were not readable on these specific examples.

Reply:
Yes, these color were re-defined in html, I think use main.css will be unified style, but I am not sure this example clean this color is better in this PR, other buffergeometry's example will be same style with other examples.

https://github.com/mrdoob/three.js/blob/e5cd04511cbdbf588b33dad2795ba2b4e5accada/examples/webgl_buffergeometry_morphtargets.html#L8-L12

Reply:
Can you share a screenshot?

Reply:
> Can you share a screenshot?

Please to see: https://github.com/mrdoob/three.js/pull/19424#issuecomment-632961213

Reply:
Thank you!